docs/usage.md

```diff
@@ -231,17 +231,15 @@ Pre-configured reference taxonomy databases are:
 | coidb        | +     | +      | -       | -      | eukaryotic Cytochrome Oxidase I (COI)         |
 | midori2-co1  | +     | -      | -       | -      | eukaryotic Cytochrome Oxidase I (COI)         |
 | standard     | -     | -      | +       | -      | any in genomes of archaea, bacteria, viruses² |
-| custom³      | +     | -      | +       | +      | any, user supplied²                           |
 
-¹: de-replicated at 85%, only for testing purposes
-²: quality of results might vary
-³: theoretically possible with all classifiers, but specific parameters only available for some tools
+¹: de-replicated at 85%, only for testing purposes; ²: quality of results might vary
```

Comment:
Number 3 is still in the table it looks like, but gone from here.

Reply:
Hm, I think 3 was only in "custom³" that I removed because its actually not a "Database key". Cannot see any other ³.